### PR TITLE
Chore guard routes

### PR DESCRIPTION
**Pull request by Midnighter**
1. This is an example of how routes can be protected based on authorization state.
2. Also replace the history of a user upon logout.

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -1,11 +1,21 @@
 import Vue from "vue";
 import Router from "vue-router";
+
```
Comment (kvikshaug) [inline src/router.ts]: I like separating the imports like this, but I don't think it makes sense to just do it ad-hoc unless we can enforce the rules with prettier/eslint/other tools. At the very least, if you want to introduce this, you should update all other imports correspondingly to stay consistent.

Comment (Midnighter) [reply src/router.ts]: The closest thing I could find was https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md but it doesn't split them. I can either revert this change or split all imports manually in a separate PR.

**Comment by Midnighter** [comment]
The only problem I couldn't solve is that it would be nice to somehow display a banner that the user needs to log in to access a denied URL.

**Comment by kvikshaug** [comment]
You could use a flag in the root store and put the template in the root app component, like we do for login messages, fetch errors etc. Note the computed property needs a setter for the vuetify component to update the `v-model` binding (if applicable).

**Pull request merged**